knexfile.js

```diff
@@ -0,0 +1,55 @@
+// Update with your config settings.
+
+module.exports = {
+
+  development: {
+    client: 'sqlite3',
+    useNullAsDefault: true,
+    connection: {
+      filename: './database/materials.db3',
+    },
+    pool: {
+      afterCreate: (conn, done) => {
+        conn.run('PRAGMA foreign_keys = ON', done);
+      },
+    },
+    migrations: {
+      directory: './database/migrations',
+    },
+    seeds: {
+      directory: './database/seeds',
+    },
+  },
+
+  testing: {//knex migrate:latest --env testing
+    client: 'sqlite3',
+    connection: {
+      filename: './database/tester.db3'
+    },
+    useNullAsDefault: true,
+    pool: {
+      afterCreate: (conn,done)=>{//make sure this is in production object too later
+        conn.run('PRAGMA foreign_keys = ON', done)
+      }
+    },
+    migrations: {
+      directory: './database/migrations'
+    },
+    seeds: {
+      directory: './database/seeds'
+    },
+  },
+
+  production: {
+    client: 'pg',
+    connection: process.env.DATABASE_URL,
+    migrations: {
+      directory: './database/migrations'
+    },
+    seeds: {
+      directory: './database/seeds/production'
```

Comment:
Are our production seeds going to be different from our development seeds?

Reply:
I wasn't sure if they were or they weren't but I think it might be a good idea if we expect to edit our default material descriptions as we get new information. Basically, we can use development seeds as a draft area since updating should be easy copy and paste.

Reply:
Okay, I'm fine with that. We can revisit this later if we change our mind.